src/modules/fancyzones/lib/JsonHelpers.h

```diff
@@ -132,6 +132,8 @@ namespace JSONHelpers
 
     struct AppZoneHistoryData
     {
+        std::map<DWORD, HWND> processIdToHandleMap;
```

Comment:
Here we need to explain what the `DWORD` primary key is.

Reply:
Is this information persisted to disk? I guess no, so will have to revisit this code later to clean it up.
A data structure that is not persisted to the json file should not belong to the JsonHelpers namespace.